src/components/Search/index.js

```diff
@@ -30,7 +35,7 @@ export const Search = () => {
         type="submit"
```

Comment:
A placeholder in the above input should also be a part of I18Y logic.

Reply:
logic also added to the placeholder